content/docs/components/jupyter.md

```diff
@@ -99,6 +99,19 @@ This means that you can directly create these k8s resources directly from your j
 ```
 !kubectl create -f myspec.yaml
 ```
+## Creating a custom Jupyter image
+You can create your own Jupyter image and use it in your Kubeflow cluster.
+Your custom image needs to meet the requirements created by Kubeflow Notebook Controller. Kubeflow Notebook Controller  manages the life-cycle of notebooks.
+ Kubeflow Web UI expects the Jupyer to be launched upon running the docker image with only `docker run`. For that you need to set the default command of your image to launch Jupyter. The Jupyter launch command needs to be set as follows:
+
+* Set the working directory: `--notebook-dir=/home/jovyan`. This is because the folder `/home/jovyan` is backed by Kubernetes Persistent Volume (PV)
+* Allow Jupyter to listen on all IPs: `--ip=0.0.0.0`
+* Allow the user to run the notebook as root: `--allow-root`
+* Set port: `--port=8888`
+* Disable authentication. Kubeflow takes care of authentication. Use the following to allow passwordless access to Jupyter: `--NotbookApp.token=''  --NotebookApp.password=''`
```

Comment:
s/NotbookApp/NotebookApp

Please test all these instructions to make sure they work.

Reply:
An example would be useful, and will also ensure that we're giving the customer working code.

Reply:
> s/NotbookApp/NotebookApp
> 
> Please test all these instructions to make sure they work.

Thanks. Already tested.